evalml/model_understanding/force_plots.py

```diff
@@ -45,12 +40,13 @@ def gen_force_plot(shap_values, training_data, expected_value, matplotlib):
         initjs()
 
     shap_plots = force_plot(pipeline, rows_to_explain, training_data, y)
-    for row in shap_plots:
+    for ix, row in enumerate(shap_plots):
```

Comment:
@peterataylor This looks good to me! Thanks for the contribution. I think the one thing that's missing is maybe a unit test to make sure we don't commit the same mistake again in the future. Maybe do one with and without a custom index on `training_data`?

Reply:
Thanks @freddyaboulton . I've added some tests to test_force_plots that use the existing parameter sets (rows 0 and 0-4). It required inspection of the returned plot object to ensure the feature values matched those that correspond to the rows_to_explain. I confirmed the tests failed with previous version and pass now. You will probably want to review though. Thanks!